s3tests_boto3/functional/test_s3.py

```diff
@@ -13166,3 +13169,108 @@ def test_multipart_upload_on_a_bucket_with_policy():
+    status, error_code = _get_status_and_error_code(e.response)
+    eq(status, 404)
+
+PUT_ORIGIN = "www.*.com"
+OTHER_ORIGIN = "*.com"
+
+ALLOWED_HEADERS = ["*"]
+ALLOWED_METHODS = ['PUT', 'POST', 'GET']
+ALLOWED_ORIGINS = [PUT_ORIGIN]
+EXPOSE_HEADERS = ['*']
+MAX_AGE_SECONDS = 123
+
+cors_config = {
+    'CORSRules': [
+        {
+            'AllowedHeaders': ALLOWED_HEADERS,
+            'AllowedMethods': ALLOWED_METHODS,
+            'AllowedOrigins': ALLOWED_ORIGINS,
+            'ExposeHeaders': EXPOSE_HEADERS,
+            'MaxAgeSeconds': MAX_AGE_SECONDS
+        },
+    ]
+}
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='get cors configuration')
+@attr(assertion='get/set cors of a bucket, returns 200')
+def test_get_cors_correct_rules():
+    bucket = get_new_bucket()
+    bucket_set_cors(bucket, cors_config)
+    response = bucket_get__cors(bucket)
+    assert response['CORSRules'][0]['AllowedMethods'] == ['GET', 'PUT']
+    assert response['CORSRules'][0]['AllowedOrigins'] == ['*']
+    assert get_response_status(response) == 200
+
+@attr(resource='bucket')
+@attr(method='delete')
+@attr(operation='delete cors configuration')
+@attr(assertion='delete cors of a bucket, returns 204')
+def test_delete_cors():
+    bucket = get_new_bucket()
+    bucket_set_cors(bucket, cors_config)
+    response = bucket_delete_cors(bucket)
+    assert get_response_status(response) == 204
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='get after delete cors configuration')
+@attr(assertion='get after delete cors of a bucket, returns 404')
+def test_get_after_delete_cors():
+    bucket = get_new_bucket()
+    bucket_set_cors(bucket, cors_config)
+    bucket_delete_cors(bucket)
+    response = bucket_get__cors(bucket)
+    assert get_response_status(response) == 404
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='get cors configuration')
+@attr(assertion='get cors by unexpected id, returns 403')
+def test_get_cors_unexpected_account_id():
+    bucket = get_new_bucket()
+    bucket_set_cors(bucket, cors_config)
+    tenant_client = get_alt_client()
+    response = bucket_get__cors(client=tenant_client, bucket)
+    assert get_response_status(response) == 403
+
+@attr(resource='bucket')
+@attr(method='set')
+@attr(operation='set cors configuration')
+@attr(assertion='set incorrect cors, fails 403')
+def test_send_not_allowed_cors():
+    bucket = get_new_bucket()
+    not_allowed_cors_config = {'Origin': OTHER_ORIGIN, 'Access-Control-Request-Method': 'GET'}
+    response = bucket_set_cors(bucket, not_allowed_cors_config)
+    assert get_response_status(response) == 403
+
+@attr(resource='bucket')
+@attr(method='set')
+@attr(operation='set cors configuration')
+@attr(assertion='set no cors, fails 403')
+def test_set_no_cors():
+    bucket = get_new_bucket()
+    no_cors_config = None
+    response = bucket_set_cors(bucket, no_cors_config)
+    assert get_response_status(response) == 200
```

Comment:
comment says 403, your python says 200, I think it should be another `400 Bad Request`